### PR TITLE
additional browser and element extensions

### PR DESCRIPTION
**Pull request by ElleryHorton**

### CODE REVIEW

File: Zukini.UI/ElementScopeExtension.cs [hunk 1]
```diff
@@ -0,0 +1,89 @@
+﻿using Coypu;
+using OpenQA.Selenium;
+using System;
+using System.Drawing;
+using System.IO;
+
+namespace Zukini.UI
+{
+    // TODO: remove once added to Zukini
```
Comment (MikeMugu) [inline Zukini.UI/ElementScopeExtension.cs]: Remove comment.

File: Zukini.UI/CoypuExtensions.cs [hunk 2]
```diff
@@ -1,8 +1,6 @@
 ﻿using Coypu;
-using OpenQA.Selenium;
 using System;
 using System.Collections.Generic;
-using System.Drawing;
 using System.Linq;
 
 namespace Zukini.UI
```
Comment (MikeMugu) [inline Zukini.UI/CoypuExtensions.cs:6]: Is this file now dead then? Should it be removed from the project?

File: Zukini.UI/Zukini.UI.csproj [hunk 3]
```diff
@@ -60,6 +60,7 @@
   <ItemGroup>
     <Compile Include="CoypuExtensions.cs" />
```
Comment (MikeMugu) [inline Zukini.UI/Zukini.UI.csproj:61]: Should CoypuExtensions.cs be removed now?

**Review by MikeMugu — CHANGES_REQUESTED** [review]

File: Zukini.UI/CoypuExtensions.cs (hunk 2 above)
Comment (ElleryHorton) [reply Zukini.UI/CoypuExtensions.cs:6]: no, CoypuExtensions still contains the table row and cells extensions where as ElementScopeExtension deals specifically with ElementScope and the navtive WebElement.

File: Zukini.UI/ElementScopeExtension.cs (hunk 1 above)
Comment (ElleryHorton) [reply Zukini.UI/ElementScopeExtension.cs]: doh, got it

File: Zukini.UI/Zukini.UI.csproj (hunk 3 above)
Comment (ElleryHorton) [reply Zukini.UI/Zukini.UI.csproj:61]: see previous comment, I think it still has a place for the Table extensions

**Review by MikeMugu — APPROVED** [review]

**Pull request merged**